src/backend/api/Fusion.Resources.Logic/Requests/Commands/ResourceAllocationRequest/JointVenture/Create.cs

```diff
@@ -0,0 +1,335 @@
+                        return requestItem!;
+                    }
+
+
+                    private async Task<DbResourceAllocationRequest> PersistChangesAsync(Create request)
+                    {
+                        var created = DateTimeOffset.UtcNow;
+
+                        var item = new DbResourceAllocationRequest
+                        {
+                            Id = Guid.NewGuid(),
+                            AssignedDepartment = request.AssignedDepartment,
+                            Discipline = request.Discipline,
+                            Type = ParseRequestType(request),
+                            State = DbResourceAllocationRequestState.Created,
+
+                            Project = Project!,
+
+                            ProposedPerson = ProposedPerson,
+                            AdditionalNote = request.AdditionalNote,
+
+                            ProposedChanges = SerializeToString(request.ProposedChanges),
+
+                            OrgPositionId = request.OrgPositionId,
+                            OrgPositionInstance = GenerateOrgPositionInstance(request.OrgPositionInstance)!,
+
+                            IsDraft = request.IsDraft,
+
+                            Created = created,
+                            CreatedBy = request.Editor.Person,
+                            LastActivity = created,
+
+                            ProposedPersonWasNotified =
+                                false, // Should be set/reset during update when/if when notifications are enabled
+
+                        };
+
+                        await db.ResourceAllocationRequests.AddAsync(item);
+                        await db.SaveChangesAsync();
+
+                        var workflow = new InternalRequestJointVentureWorkflowV1(request.Editor.Person);
+                        await db.Workflows.AddAsync(workflow.CreateDatabaseEntity(item.Id, DbRequestType.Employee));
+                        await db.SaveChangesAsync();
+
+
+                        return item;
+                    }
+
+                    private static DbResourceAllocationRequest.DbAllocationRequestType ParseRequestType(Create request)
+                    {
+                        return Enum.Parse<DbResourceAllocationRequest.DbAllocationRequestType>($"{request.Type}");
+                    }
+
+                    private static string SerializeToString(Dictionary<string, object>? properties)
+                    {
+                        var propertiesJson = JsonSerializer.Serialize(properties ?? new Dictionary<string, object>(),
+                            new JsonSerializerOptions
+                            { WriteIndented = true, PropertyNamingPolicy = JsonNamingPolicy.CamelCase });
+
+                        return propertiesJson;
+                    }
+
+                    private async Task ValidateAsync(Create request)
+                    {
```

Comment:
Tbh, the validate method here should not be required when using the fluent validation. 

I added this initially as we did not have any way of validating the commands.